gradle.properties

```diff
@@ -9,4 +9,6 @@ android.useAndroidX=true
 android.nonTransitiveRClass=true
 
 #MPP
-kotlin.mpp.enableCInteropCommonization=true
\ No newline at end of file
+kotlin.mpp.enableCInteropCommonization=true
+
+ksp.incremental=false
```

Comment:
Was this change intended?

Reply:
I removed it. I had problems with the database version and this was the first solution on the internet. Everything is working fine without it